### DISCUSSION TITLE
Feedback for 3.0.0 pre-release

### DISCUSSION THREAD

**Discussion by halo** [halo/LinkLiar#49] [General]
This is an *alpha release* for testing purposes. Please try it out. Once I feel confident enough that everything works fine and I fixed the test suite, I'll release a "real" version. Thanks for your patience.

* [BREAKING] Configuration directory moved from
             `/Library/Application Support/LinkLiar/`
             to
             `/Library/Application Suppoert/io.github.halo.LinkLiar`
             as this is more future-proof and I think the way Apple wants it.

* [FEATURE] Choose among vendors (and custom prefixes) to be used when randomizing MAC addresses #27
* [FEATURE] Do not show leakage icon for ignored interfaces #38
* [FEATURE] Show which MAC address was specified by default
* [FEATURE] Show paths to all relevant directories and files in developer menu
* [UPDATE] Update MAC prefixes of "popular" vendors

<hr /><em>This discussion was created from the release <a href='https://github.com/halo/LinkLiar/releases/tag/3.0.0'>Big Sur</a>.</em>

**Comment by berstend** [comment]
It works beautifully. I just managed to spoof my FireTV stick's mac address, sign into a pesky hotel wifi captive portal and revert the mac address (with the FireTV stick now able to connect without captive portal).

Many thanks and great job! 👍 

(M1 Air, Big Sur, `3.0.3`)

**Comment by Tanzinyt** [comment]
Hi, I need your assistance, I have followed all your instructions meticulously, and yet I am unable to load your amazing code. 
Please cal you assist me. I have Sequoia 15.5 downloaded and I have tried almost every option you have available; please do let me know.

**Comment by halo** [comment]
Hi, did you download this code?

https://github.com/halo/LinkLiar/releases/download/4.0.0.delta/LinkLiar.4.0.0.delta.zip

And do you see the LinkLiar icon in the menu bar?

Having said that, I don't have Sequoia installed yet, so it might well be that it doesn't work. Do you know how to use the Terminal? Then you could just test if you can change your MAC address at all.

**Comment by Tanzinyt** [comment]
Hi Halo,

Please can you help me, it give me this error each time I am trying to access the application.
[cid:3c28c72d-8f33-476c-80e0-293bef378c85]

I do have HomeBrew installed on my Mac OS. Please let me know how I can activate your software?
I have starred your page and I am getting a friend to install your software too; although his HomeBrew is not working.

Do you know of a developer that can help to install HomeBrew to ensure that you have an additional download; and more ratings to your great work.

Thanks,
Tanzinyt
________________________________

**Comment by Tanzinyt** [comment]
Hi Halo,

Kindly see below, I've downloaded your latest update; yet I still get the same error.
Please can you let me know what else I can.

Your assistance is greatly appreciated.

[cid:e9528200-a4c4-4ab0-a39f-6abbee61e287]

________________________________

**Comment by halo** [comment]
Homebrew has an old version right now because the beta version is not enough tested yet. So you'll have to [download](https://github.com/halo/LinkLiar/releases/download/4.0.0.delta/LinkLiar.4.0.0.delta.zip) it manually for now.

I cannot see the error you have, you'll need to walk me through it. What exactly are the steps you do before the error appears? 

- You have the zip file 
- and then you double-click on it
- you open the folder that is extracted
- you right-click on the LinkLiar app icon

Ah, I realize this might be the problem. In Sequoia you cannot right-click open any more if I remember correctly. Can you provide a screenshot of your problem?

**Comment by Tanzinyt** [comment]
Hi Hal0/LinkLiar,

Thank you so much for sharing your BETA version.
There were some settings under privacy on my MAC that I needed to fix which wasn't allowing it work.

It work's like a gem now - I will post a review on GitHub based on your excellent code.

I wanted to know if you knew of any developers that are great with Virtual Machines/installing specific Virtual Machines onto Oracle's Virtual Box on Mac? I've tried building it myself and it isn't working to be honest; I need an expert that can help me with this.

________________________________

**Comment by halo** [comment]
> It work's like a gem now
 
I'm glad you got it working.

> if you knew of any developers that are great with Virtual Machines/

Unfortunately I don't. I can recommend [UTM](https://mac.getutm.app/), [Parallels](https://www.parallels.com/) and [Docker](https://www.docker.com/). You might also try [VirtualBox](https://www.virtualbox.org/) but it's not as mature as the others.

But it really depends on what it is you're trying to achieve with the VM.

Good luck!

**Reply by Tanzinyt** [reply]
Hi Halo, 

I would greatly appreciate your recommendation as your expertise is far superior to mine based on this topic. 
I manage extremely sensitive files so privacy, security and anonymity are core fundamental checks that need to be upheld. 

In essence: I need to run a Windows OS through a Virtual Machine running on a MacOS Sequoia 15.3
The purpose is to run specific software within the Windows Virtual Machine to manage the extremely sensitive files with privacy, security and anonymity at the heart of it. 

The path is as follows: "recommended software to create Windows OS Virtual Machine" on macOS > Windows OS Virtual Machine > Software to run on Windows Virtual Machine

The recommended software needs to have full Windows OS capability with no stone left upturned. 

I have exhausted VirtualBox at length; Oracle responded back stating that they do not support macOS and are uncertain when an update can be released for macOS to run Windows VM's; based off their latest update; and recommended different hardware i.e. a PC. I cannot afford a completely different machine when a virtual machine with Windows OS can be used. 

Your recommendation would assist me greatly in this critical decision. 

Thanking you in advance Halo
